include/git2/config.h

```diff
@@ -422,6 +426,7 @@ GIT_EXTERN(int) git_config_get_string_buf(git_buf *out, const git_config *cfg, c
  * interested in. Use NULL to indicate all
  * @param callback the function to be called on each value of the variable
  * @param payload opaque pointer to pass to the callback
+ * @return non-zero to terminate the iteration.
```

Comment:
This explains how `git_config_foreach_cb` should stop callbacks, it doesn't describe the return value of the function.

```suggestion
 * @return 0 or an error code.
```